### DISCUSSION TITLE
React Native(auth): Share auth session between App and Share extension ala useUserAccessGroup

### DISCUSSION THREAD

**Discussion by Mohamed3on** [supabase/supabase#22951] [Questions]
Firebase auth has this thing called [useUserAccessGroup](https://firebase.google.com/docs/auth/ios/single-sign-on) that allows you to share user session data between different apps (like your main app and your iOS Share Extension code). 

Is there any way to do that with Supabase Auth too?

Thanks!

**Comment by ghost** [comment]
Do you have any update? 
Or a workaround?

I would be interested.
Thanks
